v21.2/restoring-backups-across-versions.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Restoring Backups Across Versions
+summary: CockroachDB's support policy for restoring backups across versions.
+toc: true
+docs_area: manage
+---
+
+This page describes the support Cockroach Labs provides for restoring backups across major versions of CockroachDB. The sections outline the support and examples for the most common use cases when restoring backups across versions:
+
+- [Support for restoring backups into a newer version](#support-for-restoring-backups-into-a-newer-version)
+- [Restore into a newer version](#restore-into-a-newer-version)
+- [Support for long-term backup archival](#support-for-long-term-backup-archival)
+
+## Support for restoring backups into a newer version
+
+Cockroach Labs supports restoring a backup on one major version following the major version on which you took the backup. Therefore, if you took a backup on major version N, you can restore it on major version N+1.
+
+For example, backups taken on v21.1 can be restored into v21.2. Backups taken on v21.2 can be restored into v22.1. The following table outlines this support:
```

Comment:
we can write these as v21.1.x, v21.2.x, to express that the minor version does not matter.

Reply:
Yes, that's a good idea. I have changed all version numbers to this pattern as I think it makes it clearer.